lms/static/sass/views/_program-details.scss

```diff
@@ -442,7 +442,7 @@
         }
 
         .course-details {
-            float: none;
+            float: left;
```

Comment:
For RTL purposes, this should be `@include float(left)`. See https://openedx.atlassian.net/wiki/spaces/LOC/overview for more info on localization (l10n). A couple RTL-specific pages are linked.

Reply:
Actually... why is the SASS for the program details page being modified?

Reply:
I didn't edit any of those files - it must've been a merging error.  I'll fix it